apps/web/src/pages/reporting.tsx

```diff
@@ -1,110 +1,25 @@
-import { useEffect, useState } from 'react';
-import dayjs from 'dayjs';
-import { writeFile } from 'xlsx-js-style';
-
-import { Access, Period } from '@ien/common';
-import { createReportWorkbook, getReportByEOI } from '@services';
-import { DataExtractReport, PageOptions, Pagination, ReportTable } from '@components';
+import { Access } from '@ien/common';
+import { DataExtractReport, ReportTable } from '@components';
 import withAuth from '../components/Keycloak';
-
-const DEFAULT_PAGE_SIZE = 10;
-const REPORT_PREFIX = 'ien-report-period';
+import { AclMask } from '../components/user/AclMask';
```

Comment:
we should start exporting these new components in the index file so we can stay consistent with @components for example